programming/typescript/challenges/index.ts

```
@@ -0,0 +1,11 @@
+// Or import the entire module
+import * as Challenges from '@lehungio/typescript';
+let helloAll = Challenges.greet('All'); // Output: "Hello, World!"
+console.log(helloAll);
+
+import { greet } from '@lehungio/typescript';
+let greetMessage = greet('Typescript'); // Output: "Hello, World!"
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'greetMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)